source/slang/slang-capability-defs.h

```diff
@@ -55,6 +55,7 @@ SLANG_CAPABILITY_ATOM0(GLSL,     glsl,      Concrete,TargetFormat,0)
 SLANG_CAPABILITY_ATOM0(C,        c,         Concrete,TargetFormat,0)
 SLANG_CAPABILITY_ATOM0(CPP,      cpp,       Concrete,TargetFormat,0)
 SLANG_CAPABILITY_ATOM0(CUDA,     cuda,      Concrete,TargetFormat,0)
+SLANG_CAPABILITY_ATOM0(SPIRV_DIRECT, spirv_direct, Concrete, TargetFormat, 0)
```

Comment:
There's some interesting questions about how capabilities like `vulkan`, `glsl`, `spirv`, and `spirv_direct` should relate. I don't think we need to resolve them right now, but it is probably necessary to resolve them in the long run in case we need overload resolution to pick between GLSL->SPIRV and direct-to-SPIRV code, while also still having some functions that are logically shared between the two cases.